### PR TITLE
Auto-save options

### LINKED ISSUE #38: Auto-save options
A "Save" button is so 2000. We can instead just auto-save when the user changes the settings by listening to the `change` event on the form inputs. This way we can also remove the alert message thingy.

### PR DESCRIPTION
**Pull request by Narno**
- listens change event on each form imput
- removes controls (« save » and « reset »)
- removes the alert message

Fixes #38.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Nice! Looks pretty good. One thing I noticed though, is that when you remove the URL it default back to `https://`. It should rather default to the default GitHub API URL.

File: extension/options.js [hunk 1]
```diff
@@ -33,35 +31,28 @@
...
-		});
-
-		formUseParticipating.addEventListener('change', function () {
-			GitHubNotify.settings.set('useParticipatingCount', formUseParticipating.checked);
+			var url = normalizeRoot(formRootUrl.value);
+			var urlSettings = normalizeRoot(formRootUrl.value) + 'settings/tokens/new?scopes=notifications';
+			// case of url is empty: set to default
+			if (url === normalizeRoot('')) {
+				GitHubNotify.settings.remove('rootUrl');
+				url = GitHubNotify.settings.get('rootUrl');
```
Comment (sindresorhus) [inline extension/options.js]: Why are you trying to get the `rootUrl` right after removing it?

Comment (sindresorhus) [inline extension/options.js]:
```js
var defaultUrl = normalizeRoot('');
if (url === defaultUrl) {
  GitHubNotify.settings.remove('rootUrl');
  url = defaultUrl;
}
```

?

Comment (Narno) [inline extension/options.js]: Because the only way to fetch the default value is the "GitHubNotify.settings.get" method. No?

Comment (sindresorhus) [inline extension/options.js]: Ah, yeah. Makes sense.

**Comment by sindresorhus** [comment]
Looks good. Thanks :)

**Pull request closed without merging**